tasks/pipeline.py

```diff
@@ -249,6 +250,10 @@ def run(
     Use --no-kitchen-tests to not run all kitchen tests on the pipeline.
     Use --e2e-tests to run all e2e tests on the pipeline.
 
+    Release Candidate related flags:
+    Use --rc-build to mark the build as Release Candidate.
+    Use --rc-k8s-deployments to trigger a child pipeline that will deploy Release Candidate build to staging k8s clusters (only with --rc-build)
```

Comment:
> (only with --rc-build)
I don't see this logic reflected in the code, am I missing something?

Reply:
It's not reflected in the code and it actually does not make much sense to enforce that. I will remove that sentence.